src/main/groovy/com/github/lkishalmi/gradle/gatling/LogbackConfigTaskAction.groovy

```diff
@@ -23,10 +39,21 @@ public class LogbackConfigTaskAction implements Action<Task> {
     void execute(Task gatlingRunTask) {
         def gatlingExt = gatlingRunTask.project.extensions.getByType(GatlingPluginExtension)
         if (!gatlingRunTask.project.file("${gatlingExt.confDir()}/logback.xml").exists()) {
-            new File(gatlingRunTask.project.buildDir, "resources/gatling/logback.xml").with {
-                parentFile.mkdirs()
-                text = template(gatlingExt.logLevel)
+
+            // gatling 2.3.x updated logback and the config file has changed therefore we must differentiate between >= 2.3 and <2.3
+            // see https://github.com/gatling/gatling/commit/6a35dbaa8607deda4a20abf1e8a553d9cd9cef01 for details
+            if (gatlingExt.toolVersion.startsWith("2.3")) {
```

Comment:
Maybe it is better to do a check for a number as this becomes quite hard to maintain when gatling updates to 2.4 for example.

Reply:
Hmm, I believe you may ignore that completely.